less/home-page.less

```diff
@@ -1 +1,130 @@
-// Home Content
\ No newline at end of file
+// Home Content
+.content-large {
+    width: 100%;
+}
+
+.content-medium {
+    width: 48%;
+
+    @media (@phone) {
+        width: 100%;
+    }
+}
+
+.content-small {
+    width: 31%;
+
+    @media (@phone) {
+        width: 84%;
+    }
+}
+
+.container {
+    max-width: 900px;
+    width: 100%;
+    margin: 0px auto;
+}
+
+.mobile-hide {
+    @media (@phone) {
+        display: none;
+    }
+}
+
+main {
+    .flex-column-center;
+
+    .home-image {
+        margin-bottom: 20px;
+
+        @media (@phone) {
+            margin-bottom: 15px;
+        }
+    
+        img {
+            height: 225px;
+
+            @media (@phone) {
+                height: 200px;
+            }
+        }
+    }
+
+    .text-container {
+        width: 95%;
+
+        @media (@phone) {
+            width: 93%;
+        }
+    }
+
+    .image-container {
+        width: 95%;
+        .flex-space-between;
```

Comment:
Even something like this. It's still readable and I know what you are doing here